tests/slow/src/test/scala/tests/feature/DefinitionCrossLspSuite.scala

```diff
@@ -59,8 +56,9 @@ class DefinitionCrossLspSuite
            |}
            |""".stripMargin
       )
-      _ = server.didOpen("a/src/main/scala/a/Main.scala")
-      _ = server.didOpen("a/src/main/scala/a/Test.scala")
+      _ <- server.didOpen("a/src/main/scala/a/Main.scala")
+      _ <- server.didOpen("a/src/main/scala/a/Test.scala")
+      _ = assertNoDiagnostics()
```

Comment:
That's one of the reasons why I'm always complaining about `_ = ...` in for comprehensions :)